Objects/regions.c

```diff
@@ -160,22 +196,63 @@ typedef struct _gc_runtime_state GCState;
 #define GC_PREV _PyGCHead_PREV
 #define FROM_GC(g) ((PyObject *)(((char *)(g))+sizeof(PyGC_Head)))
 
+static int
+is_immutable_region(regionmetadata* r)
+{
+    return ((Py_uintptr_t) r) == _Py_IMMUTABLE;    
```

Comment:
There is a linter that complains about extra spaces at the end of a line.
```suggestion
    return ((Py_uintptr_t) r) == _Py_IMMUTABLE;
```